op-challenger/game/fault/types/position.go

```diff
@@ -25,13 +25,27 @@ func NewPosition(depth int, indexAtDepth *big.Int) Position {
 	}
 }
 
+// NewPositionFromGIndex creates a new Position given a generalized index,
+// numbered like the following:
+//
+//			 1
+//		    /  \
+//	      2     3
+//		 / \   / \
+//		4   5 6   7
```

Comment:
I think I'd be tempted to move this ascii art into the spec and then just have the link here.

Reply:
The text should be sufficient for documentation purposes, would suggest to remove the link and just keep it simple like so:
```suggestion
// NewPositionFromGIndex creates a new Position given a generalized index.
```